package.json

```diff
@@ -1,11 +1,30 @@
 {
   "name": "@divriots/dockit-react",
-  "version": "0.0.66",
+  "version": "0.0.67",
   "description": "",
   "license": "ISC",
   "type": "module",
   "homepage": "https://backlight.dev/preview/wzPn6NO54tDJlBxCtjGx/",
   "scripts": {
     "serve": "npx backlight@latest serve wzPn6NO54tDJlBxCtjGx --open"
+  },
+  "devDependencies": {
+    "chai": "latest",
+    "twind": "latest"
+  },
+  "dependencies": {
+    "@mdx-js/react": "latest",
+    "copy-to-clipboard": "latest",
+    "dependency-graph": "latest",
+    "lodash": "latest",
+    "path": "latest",
+    "prism-react-renderer": "latest",
+    "prop-types": "latest",
+    "react": "latest",
+    "react-dom": "latest",
+    "react-live": "latest",
+    "styled-components": "latest",
+    "styled-system": "latest",
+    "tailwindcss": "latest"
```

Comment:
you shouldn't use `latest` here, I should have been more clear in Discord

you should pin the latest, meaning you find which actual number is latest and pin `^x.y.z`

Reply:
Got it, that's what I wasn't understanding regarding the version. Thanks!